src/components/Error/index.js

```diff
@@ -1,28 +1,51 @@
-import { h } from 'preact'
+import { h, Component } from 'preact'
 import classNames from 'classnames'
 import errors from '../strings/errors'
 import style from './style.css'
 import { identity, noop } from 'components/utils/func'
 import { localised } from '../../locales'
 
-const Error = ({className, error, translate, withArrow, renderMessage = identity, renderInstruction = identity, isDismissible, onDismiss = noop}) => {
-  const { message, instruction } = errors[error.name]
-  const errorType = error.type === 'error' ? 'error' : 'warning'
-  return (
-    <div className={classNames(style[`container-${errorType}`], className)}>
-      { withArrow && <div className={classNames(style.roundedTriangle, style[`${errorType}Triangle`])} /> }
-      <div className={style.title}>
-        <span className={style[`title-icon-${errorType}`]}/>
-        <span className={style['title-text']}>{renderMessage(translate(message))}</span>
+class Error extends Component {
+  componentDidMount() {
+    if (this.props.focusOnRender && this.container) {
```

Comment:
Would this work by just using `autofocus=true` in the parent div?

Reply:
No, regular `autoFocus` does not move the focus of tabbing order, that's why you need to hard-focus on the element.